src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor

```diff
@@ -39,56 +44,68 @@
     {
         DumpEvent(e);
         message += "onmouseover,";
-        StateHasChanged();
```

Comment:
None of these `StateHasChanged` calls should have been there. They're just not needed.